### PR TITLE
Add nodejs-es6-promise

### PR DESCRIPTION
**Pull request by dLobatog**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.12
* [ ] 1.11
* [ ] 1.10

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

Needed for https://github.com/theforeman/foreman/pull/3736/files

### CODE REVIEW

**Comment by dLobatog** [comment]
Don't merge yet, I just realised I have to add this to Foreman too

File: rel-eng/tito.props [hunk 1]
```diff
@@ -23,6 +23,7 @@ whitelist = foreman-installer
   nodejs-babel-loader
   nodejs-babel-preset-es2015
   nodejs-expose-loader
+  nodejs-es6-promise
```
Comment (domcleal) [inline rel-eng/tito.props]: Swap expose/es6 to be in alphabetical order.

File: comps/comps-foreman-fedora24.xml [hunk 2]
```diff
@@ -37,6 +37,7 @@
       <packagereq type="default">nodejs-babel-loader</packagereq>
       <packagereq type="default">nodejs-babel-preset-es2015</packagereq>
       <packagereq type="default">nodejs-expose-loader</packagereq>
+      <packagereq type="default">nodejs-es6-promise</packagereq>
```
Comment (domcleal) [inline comps/comps-foreman-fedora24.xml]: ditto, swap lines to be alphabetical please

**Comment by domcleal** [comment]
I think this is failing to run the npm install for dependencies correctly, please check the registry file is working correctly.

**Comment by domcleal** [comment]
Merged as 59e15de with comments fixed, thanks @dLobatog.

**Pull request closed without merging**